UnityProject/Assets/Scripts/Items/Implants/Organs/Eye.cs

```diff
@@ -55,6 +46,17 @@ public override void Awake()
 			UpdateBlurryEye();
 		}
 
+		public override void OnAddedToBody(LivingHealthMasterBase livingHealth)
```

Comment:
remember not to confuse as between gaining authority over body versus possessing and unpossessing at the moment

Reply:
oh sortry got confused between mind possessing and body part  adding